pkg/services/cosmosdb/catalog.go

```diff
@@ -9,16 +9,16 @@ func (m *module) GetCatalog() (service.Catalog, error) {
 			service.NewService(
 				&service.ServiceProperties{
 					ID:   "6330de6f-a561-43ea-a15e-b99f44d183e6",
-					Name: "azure-cosmos-document-db",
-					Description: "Azure DocumentDB (Experimental) provided by CosmosDB " +
-						"and accessible via SQL (DocumentDB), Gremlin (Graph), and Table " +
-						"(Key-Value) APIs",
+					Name: "azure-cosmos-db-sql",
```

Comment:
I know you opted not to change the name just now, but I think we really need to be clear about _what_ each service provisions. Any thoughts on going with `azure-cosmosdb-sql-account`? (Note also that in addition to qualifying this with "account," I'm proposing to drop the "-" between "cosmos" and "db". I think it's unfortunate that the service has "DB" in its name as that can lead to some confusion. [For the same reason, I dislike "SQL Server" having "server" in its name.] By dropping the "-" between "cosmos" and "db", I hope to emphasize that this is a product name and _not_ mislead anyone into thinking that what they are provisioning is a database and not an account.)

Stop me if I'm talking crazy.

Reply:
nope, that's cool.